src/scripts/main.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+
+  if (!e.target.closest('.wall')) {
+    return;
+  };
+
+  const limitWidth = wall.clientWidth - spider.clientWidth;
+  const limitHeight = wall.clientHeight - spider.clientHeight;
+  let x = e.offsetX - spider.clientWidth / 2;
```

Comment:
`Math.max(0, e.offsetX - spider.clientWidth / 2)` - if you add this, you can delete this if.
![image](https://user-images.githubusercontent.com/56752562/224286793-f181cab3-9bac-4fbb-9868-77a86315ddbf.png)

Reply:
Same situation with `y`

Reply:
You can wrap it with `Math.min` and remove this condition.
![image](https://user-images.githubusercontent.com/56752562/224287261-5788e901-1229-4113-b02f-fd6c0eea49cc.png)

Reply:
Same situation with `y`